### DISCUSSION TITLE
could not connect to FTP server

### DISCUSSION THREAD

**Discussion by Dushku** [hippie68/ftpdump#22] [Q&A]
I have taken the ftp.bin and put it on my PS4 at /data/payloads.

Installed Debian on my 64-bit Win 10. Installed Wget and cURL.

Rebooted my PS4. Fresh jailbreak. Used Payload Guest to run ftp.bin and taken note of the IP and port. Launched Shadow of the Tomb Raider and waited for title screen.

In cmd, typed `ftpdump 192.168.1.200:1337` It says could not connect to FTP server.

I have a USB port going bad on my PS4, so I really want to get this working. Any advice?

[EDIT]
Wanted to add that while trying to research this, I saw somebody used the GoldHEN FTP server. When I turned that on and tried this (with the adjusted port), it started working. I also saw that somebody had loaded your ftp.bin from their PS4 browser. However, when I got to [https://hippie68.github.io/702/index.html](url), it says it couldn't connect. I'm using manual DNS of 192.241.221.79 and 165.227.83.145

[EDIT 2]
Once I realized that the 702 in the URL was for a different firmware, I went to change it to 900. On my PS4, the site actually came up! But then 900 does not and 702 anymore either. Should still work if I just manually launch you ftp.bin though, right?

**Comment by hippie68** [comment]
I don't have a 9.00 jailbreak website, but sending the FTP payload manually should always work. It may also be possible to use the 7.xx websites as payload launchers for 9.00 if GoldHEN's BinLoader server is enabled. Most of their payloads, including the FTP payload, should be compatible with 9.00.
